twig/twig/CVE-2025-24374.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Missing output escaping for the null coalesce operator
+link:      https://symfony.com/blog/twig-cve-2025-24374-missing-output-escaping-for-the-null-coalesce-operator
+cve:       CVE-2025-24374
+branches:
+    3.x:
+        time:     2025-01-29 07:52:00
```

Comment:
The time looks off. Shouldn't this be UTC? GitHub shows a commit time of 6:52 AM GMT/UTC

Reply:
yes, you are right about it 👍